modules/coreboot

```diff
@@ -110,11 +112,13 @@ coreboot_dir := $($(coreboot_module)_dir)
 coreboot_base_dir := $($(coreboot_module)_base_dir)
 
 $(coreboot_module)_depends += $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
-$(coreboot_module)_depends += $(if $(CONFIG_NITROKEY_BLOBS), nitrokey-blobs)
 
 # coreboot builds are specialized on a per-target basis.
 # The builds are done in a per-target subdirectory
 CONFIG_COREBOOT_CONFIG ?= config/coreboot-$(BOARD).config
+CONFIG_COREBOOT_LOCALVERSION ?= $(BRAND_NAME)-$(HEADS_GIT_VERSION)
+CONFIG_COREBOOT_SMBIOS_PRODUCT_NAME ?= $(BOARD)
+CONFIG_COREBOOT_SMBIOS_MANUFACTURER ?= $(BRAND_NAME)
```

Comment:
It was not being overridden by heads before, so not sure what the default should be. I would skip overriding if `CONFIG_COREBOOT_SMBIOS_MANUFACTURER` is not set.

Reply:
@miczyg1 Okay, done in https://github.com/Dasharo/heads/pull/2/commits/6a7ee7f6a4b5a9d2c1ff49a14884396d84629b67